guides/theming.md

```diff
@@ -367,6 +367,83 @@ $my-palette: mat.define-palette(mat.$indigo-palette);
 }
 ```
 
+## Strong focus indicators
+
+By default, most components indicate browser focus by changing their background color as prescribed
+by the Material Design specification. This behavior, however, can fall short of accessibility
+requirements, such as [WCAG][], which require a stronger indication of browser focus.
+
+Angular Material supports rendering highly visible outlines on focused elements. Applications can
+enable these strong focus indicators via two Sass mixins:
+`strong-focus-indicators` and `strong-focus-indicators-theme`.
+
+The `strong-focus-indicators` mixin emits structal indicator styles for all components. This mixin
+should be included exactly once in an application, similar to the `core` mixin described above.
+
+The `strong-focus-indicators-theme` mixin emits only the indicator's color styles. This mixin should
+be included once per theme, similar to the theme mixins described above.
```

Comment:
I believe that the mixin also needs to be included in cases where the background color is not contrastive against the focus indicator's color. The example that comes to mind is `mat-toolbar`.

Reply:
Good catch, added a sentence mentioning this.